app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -70,5 +72,19 @@ def rss_notification_blueprint
     def notification_already_exists?(item)
       !!Notification.unscoped.find_by_message(item.summary)
     end
+
+    def load_rss_feed
+      uri = URI.parse(@url)
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
```

Comment:
The documentation at https://ruby-doc.org/stdlib-2.4.2/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-HTTPS suggests this is no longer needed if you just pass in the uri rather than the ```uri.request_uri```.